test/geometry/lane_geometry_test.cc

```diff
@@ -107,58 +107,81 @@ std::vector<OrientationTestCase> OrientationTestCases() {
           {{0., -M_PI_4, -M_PI_4}, {0., -M_PI_4, -M_PI_4}} /* expected_rpy */
       },
       {
-          // Increase elevation + plateau + decrease elevation.
+          // #4 - Increase elevation + plateau + decrease elevation.
           LineString3d{{0., 2., 0.}, {10., 2., 10.}, {20., 2., 10.}, {30., 2., 0.}} /* left*/,
           LineString3d{{0., -2., 0.}, {10., -2., 10.}, {20., -2., 10.}, {30., -2., 0.}} /* right*/,
           std::nullopt /* center*/,
           {0., 1 + std::sqrt(2.) * 10., 1 + std::sqrt(2.) * 10. + 10.} /* p */,
           {{0., -M_PI_4, 0.}, {0., 0., 0.}, {0., M_PI_4, 0.}} /* expected_rpy */
       },
       {
-          // Constant superelevation to the left.
+          // #5 - Constant superelevation to the left.
           LineString3d{{0., 2., 0.}, {100., 2., 0.}} /* left*/,
           LineString3d{{0., -2., 4.}, {100., -2., 4.}} /* right*/,
           std::nullopt /* center*/,
           {0., 100.} /* p */,
           {{-M_PI_4, 0., 0.}, {-M_PI_4, 0., 0.}} /* expected_rpy */
       },
       {
-          // Constant superelevation to the right.
+          // #6 - Constant superelevation to the right.
           LineString3d{{0., 2., 4.}, {100., 2., 4.}} /* left*/,
           LineString3d{{0., -2., 0.}, {100., -2., 0.}} /* right*/,
           std::nullopt /* center*/,
           {0., 100.} /* p */,
           {{M_PI_4, 0., 0.}, {M_PI_4, 0., 0.}} /* expected_rpy */
       },
+  };
+}
+
+class OrientationTest : public ::testing::TestWithParam<OrientationTestCase> {
+ public:
+  OrientationTestCase case_ = GetParam();
+
+  virtual double GetTolerance() const {
+    static constexpr double kTolerance{1e-12};
+    return kTolerance;
+  }
+
+  void ExecuteOrientationTest() {
+    ASSERT_EQ(case_.p.size(), case_.expected_rpy.size()) << ">>>>> Test case is ill-formed.";
+    const LaneGeometry lane_geometry = case_.center.has_value()
+                                           ? LaneGeometry{case_.center.value(), case_.left, case_.right, 1e-3, 1.}
+                                           : LaneGeometry{case_.left, case_.right, 1e-3, 1.};
```

Comment:
Please, convert 1e-3 and 1. into named constants.

Reply:
Done.